e2e/cypress/tests/integration/enterprise/extend_session/email_login_activity_spec.ts

```diff
@@ -44,7 +46,8 @@ describe('MM-T2575 Extend Session - Email Login', () => {
                 ExtendSessionLengthWithActivity: true,
                 SessionLengthWebInHours: 1,
             },
-        };
+        } as AdminConfig;
```

Comment:
Please use [the one under `Cypress` namespace](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/index.d.ts#L7) so it's imported in one place only.

```suggestion
        } as Cypress.AdminConfig;
```

Reply:
Fixed.